src/board/system76/galp5/board.c

```diff
@@ -38,48 +37,9 @@ void board_init(void) {
     SPCTRL1 |= 0xC8;
 }
 
-#if HAVE_DGPU
-// Set PL4 using PECI
-static int set_power_limit(uint8_t watts) {
-    return peci_wr_pkg_config(
-        60, // index
-        0, // param
-        ((uint32_t)watts) * 8
-    );
-}
-
-void board_on_ac(bool ac) {
-    uint8_t power_limit = ac ? POWER_LIMIT_AC : POWER_LIMIT_DC;
-    // Retry, timeout errors happen occasionally
-    for (int i = 0; i < 16; i++) {
-        int res = set_power_limit(power_limit);
-        DEBUG("set_power_limit %d = %d\n", power_limit, res);
-        if (res == 0x40) {
-            break;
-        } else if (res < 0) {
-            ERROR("set_power_limit failed: 0x%02X\n", -res);
-        } else {
-            ERROR("set_power_limit unknown response: 0x%02X\n", res);
-        }
-    }
-}
-#else // HAVE_DGPU
-void board_on_ac(bool ac) { /* Fix unused variable */ ac = ac; }
-#endif // HAVE_DGPU
-
 void board_event(void) {
 #if HAVE_DGPU
-    static bool last_power_limit_ac = true;
-    // We don't use power_state because the latency needs to be low
-    if (gpio_get(&CPU_C10_GATE_N)) {
```

Comment:
ESPI boards must use CPU_C10_GATE_N, ensure this is handled correctly

Reply:
Done